packages/stateless/src/lib/provider/provider.interface.ts

```diff
@@ -28,7 +28,7 @@ export interface IEmailOptions {
 
 export interface ISmsOptions {
   to: string;
-  content: string;
+  content: any;
```

Comment:
I don't agree with that change - it should be a string and is expected to be string by other providers.